src/Stratis.Bitcoin.Features.Wallet/WalletManager.cs

```diff
@@ -390,15 +390,15 @@ public Wallet RecoverWallet(string name, ExtPubKey extPubKey, int accountIndex,
             return wallet;
         }
 
-        private HdAccount AddNewAccount(ExtPubKey extPubKey, int accountIndex, Wallet wallet)
+        private HdAccount AddNewAccountLock(ExtPubKey extPubKey, int accountIndex, Wallet wallet)
```

Comment:
AddNewAccountLock => AddNewAccountLocked

Reply:
I just copied the pattern of the other ones in the file which had Lock

Reply:
Ok, didn't see them. 

Please change other ones as well to have Locked in the end (that's the standard naming policy for methods that should be locked and it is used everywhere in the codebase. Looks like this class got inconsistent)